parser.ts

```diff
@@ -310,7 +346,7 @@ export function traverseStmt(c : TreeCursor, s : string) : Stmt<null> {
       while(c.nextSibling()) {
         body.push(traverseStmt(c, s));
       }
-      c.parent(); 
+      c.parent();
```

Comment:
Was this a bug?

Reply:
Oooohhh I see there's lots of whitespace changes. This is going to cause some annoying conflicts. Is it possible for you to turn this off *for now* and we'll then deploy an automatic formatter after we merge proposals?

Reply:
Resolved